app/views/labs/_form.html.haml

```diff
@@ -167,20 +167,19 @@
       %strong Add Capabilities:
       %span Check all of the boxes that apply to this lab
 
-    .form-check.mb-0.mt-3
+    .form-check.mb-3.pb-3
       = f.input :capabilities, as: 'check_boxes', inline_label: true, item_wrapper_class: 'checkbox_container mr-3', label: false, collection: Lab::Capabilities.map{|f| [ I18n.t("capabilities.#{f}"), f]}, checked: f.object.capabilities, input_html: { class: 'form-check-input' }
 
-    - @collection = Machine.includes(:brand)
-    - @selected = @lab.machines.map{|u| u.id}.collect
-
     - if lab.new_record?
-      %p You will be able to add individual machines once your lab is approved.
+      .mb-3.pb-3
+        %strong You will be able to add individual machines once your lab is approved.
     - else
       %label{for: "facilities_attributes"}
-        Machine
-        = select_tag :facilities_attributes, options_from_collection_for_select(@collection, :id, :name, @selected), { multiple: true, size: 10, name: 'lab[facilities_attributes][][thing_id]', class: 'machine_selection_attributes m-5 p-5'}
+        %h5
+          %strong Add Machines:
+      .mt-3.px-0.mx-4
+        = f.association :machines, wrapper_html: { class: 'w-100 overflow-hidden' }
```

Comment:
But why do we need helper classes. When you use select2, it does not overflow. Why is it overflowing on our app? What is the root problem?
It's not good to always add more classes to fix a problem, better to solve the core problem.
Adding more CSS classes should be the last option.

Reply:
I followed this: https://github.com/select2/select2/issues/2984. There are 2 solutions, 1 is to add overflow-hidden CSS and the 2nd one is not working :-? Do you have any hint of what else could be a problem?

Reply:
This is from another 'fresh' project with no custom CSS, where all the boxes align nicely and the form expands when it reaches another line.
Our old CSS is probably causing it to not expand, and it would be nice if we can fix it, because if we do, we will have fixed it for ALL select boxes.
There might be many select boxes in the codebase, and manually having to add CSS classes all of them is not the practical way.

![2020-04-15_09-14-54](https://user-images.githubusercontent.com/1689020/79308900-aa756c00-7ef9-11ea-990a-51687308524f.png)

Reply:
I removed custom CSS and it works!
<img width="503" alt="Screen Shot 2020-04-16 at 2 18 15 PM" src="https://user-images.githubusercontent.com/43182178/79455669-85622580-7fed-11ea-8ea7-2e4f4ebcf24c.png">